### PR TITLE
Update/identity view

### PR DESCRIPTION
**Pull request by ag12**
# Why?
We want to hide the profile for anonym users. 

# What?
Added offline display mode for Identity view. 

# UI Changes
Simulator Screen Shot - iPhone 11 Pro - 2020-04-08 at 15 38 39

### CODE REVIEW

**Comment by pimms** [comment]
🥇

**Comment by ag12** [comment]
Simulator Screen Shot - iPhone 11 - 2020-05-11 at 14 08 02

File: FinniversKit/Sources/Components/Profile/IdentityView.swift [hunk 1]
```diff
@@ -223,22 +244,31 @@ public class IdentityView: UIView {
...
+
+        } else if viewModel.displayMode == .anonymous {
             subtitleLabel.isHidden = true
         } else {
             subtitleLabel.isHidden = false
             subtitleLabel.text = viewModel.subtitle
         }
 
-        let showDescription = viewModel.description != nil && !hideDescription
+        let showDescription = ((viewModel.description != nil && !hideDescription) && viewModel.displayMode != .offline)
```
Comment (robinsalehjan) [inline FinniversKit/Sources/Components/Profile/IdentityView.swift:271]: Jesus

**Comment by ag12** [comment]
This PR contains BREAKING Changes.

**Pull request merged**